cmd/minikube/cmd/start_flags.go

```diff
@@ -253,17 +253,14 @@ func generateClusterConfig(cmd *cobra.Command, existing *config.ClusterConfig, k
 		klog.Info("no existing cluster config was found, will generate one from the flags ")
 		cc = generateNewConfigFromFlags(cmd, k8sVersion, drvName)
 
-		cnm, err := cni.New(cc)
+		cnm, err := cni.New(&cc)
 		if err != nil {
 			return cc, config.Node{}, errors.Wrap(err, "cni")
 		}
 
 		if _, ok := cnm.(cni.Disabled); !ok {
 			klog.Infof("Found %q CNI - setting NetworkPlugin=cni", cnm)
 			cc.KubernetesConfig.NetworkPlugin = "cni"
-			if err := setCNIConfDir(&cc, cnm); err != nil {
```

Comment:
The pr description says only for Crio we set the conf dir, here I see it removed regardless of runtime, is this right ?

Reply:
this was moved to cni package and replaced with configureCNI func, where only for crio we set cni.Network (that gets passed to crio by setting the 'cni_default_network' config param, and no other modifications are needed for crio)

for other runtimes cni.configureCNI sets the kubelet.cni-conf-dir to '/etc/cni/net.mk' value via cni.ConfDir